src/components/DateRangePicker.jsx

```diff
@@ -238,6 +241,11 @@ class DateRangePicker extends React.PureComponent {
     });
   }
 
+  onDayPickerFocusOut(event) {
+    if (this.dayPickerContainer.contains(event.relatedTarget || event.target)) return;
```

Comment:
@majapw I'll add a comment to explain this one better, but it fixes the issue where clicking on one of the days caused the `DayPicker` to close. In this case, it will set **relatedTarget** to `null`, while in the other cases it gives us the correct **relatedTarget** but an incorrect **target**.